src/plays/lorem-generator/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Lorem Generator
+
```

Comment:
Hi @Madflows 

Please provide some more details in the Readme. Consider this as one of the document a beginner will read to get the technical details of your play.